packages/backend-tasks/src/tasks/TaskWorker.ts

```diff
@@ -123,23 +126,46 @@ export class TaskWorker {
   /**
    * Perform the initial store of the task info
    */
-  async persistTask(settings: TaskSettingsV1) {
+  async persistTask(settings: TaskSettingsV2) {
     // Perform an initial parse to ensure that we will definitely be able to
     // read it back again.
-    taskSettingsV1Schema.parse(settings);
+    taskSettingsV2Schema.parse(settings);
+
+    let startAt: DateTime = settings.initialDelayDuration
+      ? DateTime.now().plus(Duration.fromISO(settings.initialDelayDuration))
+      : DateTime.now();
+
+    const isCron = !settings?.cadence.startsWith('P');
+
+    // if we don't have a delay setting, and this is a crontab configuration, then the next start time is simply the
+    // time cron should trigger he job.
+    if (!settings.initialDelayDuration && isCron) {
+      const n = settings as TaskSettingsV2;
+      const time = new CronTime(n.cadence).sendAt().toISOString();
+      // start at the next cadence, sub one second to call the function immediately if we're using "* * * * * *"
+      startAt = DateTime.fromISO(time).minus(
+        Duration.fromObject({ seconds: 1 }),
+      );
+    }
+
+    const startRun =
+      this.knex.client.config.client === 'sqlite3'
+        ? this.knex.raw('datetime(?)', [startAt.toISO()])
+        : this.knex.raw(`?`, [startAt.toISO()]);
+
+    this.logger.debug(
+      `version: ${settings.version} starting task at: ${startAt}`,
+    );
```

Comment:
remove

Reply:
Yeah 😅 this line isn't very useful now that we're aligned on the single version for use. 

For personal future reference, does this community (perhaps the Node community as a whole?) not tend to leave in debug lines for debugging issues in deployments? I'd love to rewrite this log to show when the next start time is for this ID if not

Reply:
Ah sure, that message might be useful long term too, so could keep that I guess

Reply:
But on a more long form detailed message perhaps, doesn't have to contain the version probably for example if we only handle one version at a time anyway etc

Reply:
Absolutely, updated to a more helpful message, though not super long for any need:
```
    this.logger.debug(`task: ${this.taskId} configured to run at: ${startAt}`);
```

Also adjusted the others in tryReleaseTask to:

```
      this.logger.debug(`task: ${this.taskId} will next occur around ${time}`);
```